test/tools/libtesteth/TestHelper.cpp

```diff
@@ -394,21 +406,7 @@ void checkHexHasEvenLength(string const& _str)
 
 bytes importCode(json_spirit::mObject const& _o)
 {
-    bytes code;
-    if (_o.count("code") == 0)
-        return code;
-    if (_o.at("code").type() == json_spirit::str_type)
-        if (_o.at("code").get_str().find("0x") != 0)
-            code = fromHex(compileLLL(_o.at("code").get_str()));
-        else
-            code = importByteArray(_o.at("code").get_str());
-    else if (_o.at("code").type() == json_spirit::array_type)
-    {
-        code.clear();
-        for (auto const& j : _o.at("code").get_array())
-            code.push_back(toByte(j));
-    }
-    return code;
+    return processDataOrCode(_o, "code");
```

Comment:
now why do we need importCode function?

Reply:
Feel free to do it - this PR tried to keep changes at minimum.